src/Jellyfin.XmlTv/XmlTvReader.cs

```diff
@@ -409,7 +409,7 @@ public void ProcessStarRating(XmlReader reader, XmlTvProgram result)
             if (reader.Name == "value")
             {
                 var textValue = reader.ReadElementContentAsString();
-                int index = textValue.IndexOf('/');
+                int index = textValue.IndexOf('/', StringComparison.InvariantCulture);
```

Comment:
StringComparison.Ordinal has better performance

Reply:
I had Ordinal at first, but I switched it to this because I saw it elsewhere. I'll change it.